base/linalg/arnoldi.jl

```diff
@@ -349,9 +349,9 @@ X = sprand(10, 5, 0.2)
 svds(X, nsv = 2)
 ```
 
-**Implementation note**
+!!! Implementation note
```

Comment:
Should be `!!! note "Implementation"` if you'd like a `note` admonition with the title "Implementation".